docs/get-started/alerting.md

```diff
@@ -8,59 +8,101 @@ Alerting notifies of important or unusual activity in your database environments
 For example, you can create a silence to suppress trivial notifications during weekends.
 
 ### Contact points
-Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, a notification is sent to the specified contact points. 
 
-Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues. 
+Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, a notification is sent to the specified contact points.
 
-You can choose from a variety of contact points, including Slack, email, webhooks, PagerDuty, and more. 
+Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues.
+
+You can choose from a variety of contact points, including Slack, email, webhooks, PagerDuty, and more.
 
 ### Notification policies
-Notification policies determine how Grafana alerts are routed to contact points by setting where, when, and how to send notifications. 
+
+Notification policies determine how Grafana alerts are routed to contact points by setting where, when, and how to send notifications.
 
 For example, you might specify a limit for the number of times a notification is sent during a certain period. This helps ensure that you don't spam your Slack channel with too many notifications about the same issue.
 
+#### Configure Email (SMTP) server settings
+Percona Alerting comes pre-configured with a Notification Root Policy, which is the default notification policy that PMM applies to ALL alerts that don’t have a custom notification policy assigned to them. This root notification policy is called **grafana-default-email** and uses Email as the default contact point.
+
+To configure Email notifications for the root policy or any custom Email policies, edit the PMM’s **grafana.ini** file. This holds the settings for [Grafana’s Alertmanager](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/alertmanager/) which routes notifications to contact points.
+
+To use SMTP with a PMM Docker installation:
+
+1. Create an `.env` file and populate it with your SMTP credentials (and other environment variables) as follows:
+
+        ```
+        GF_SMTP_ENABLED=true  
+        GF_SMTP_HOST=smtp.gmail.com:587
+        GF_SMTP_USER=email@domain.com
+        GF_SMTP_PASSWORD=<YOUR_SMTP_PASSWORD>
+        GF_SMTP_SKIP_VERIFY=false
+        GF_SMTP_FROM_ADDRESS=email@domain.com
+        GF_SMTP_FROM_NAME=Percona Alerting
+        ```
+
+2. Pass in the `.env` file to Docker run using the `--env-file` flag: ```docker run --env-file=.env -p 443:443 -p 80:80 percona/pmm-server:2```.
+
+3. In the grafana.ini file, add the following section and lines:
```

Comment:
Using 3 as an alternative to 1 is not advisable since it may not be compatible with upgrades. It should only be utilized for AMI/ova purposes.

Reply:
@rnovikovP I guess in this case, we can remove number 3 completely since the context here is docker

Reply:
and 4. can be removed